src/components/navbar.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export default function Navbar() {
+  return (
+    <div className="navbar">
+      <h1 className="navTitle">Magician Calculator!</h1>
+      <nav className="navLinks">
+        <Link className="navItem" to="/">Home</Link>
+        {' '}
+        |
+        {' '}
+        <Link className="navItem" to="/aboutCalculator">Calculator</Link>
```

Comment:
- [ ] @sajjadrajaee great work on your project, kindly ensure your link is correctly labeled, your calculator link is supposed to display the calculator but it's displaying your about calculator page.

Reply:
Thank you @danmainah for your time and review of my code. I will fix it. 👍